src/otx/algo/detection/utils/samplers/__init__.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Samplers for detection task."""
+
+from .base_sampler import PseudoSampler, RandomSampler
```

Comment:
Do you think these samplers can only be used for detection?

Reply:
I moved all modules that are used across tasks (det and iseg) to `common` directory (https://github.com/openvinotoolkit/training_extensions/pull/3636/commits/27e15dd35e328719025fbedd09dd75f035765db5).